explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -146,6 +149,20 @@ export function UpgradeableProgramSection({
             )}
           </td>
         </tr>
+        <tr>
+          <td>
+            {
+              //TODO: display also for non upgradeable programs
```

Comment:
The explorer currently only saves the program data, if it can be decoded, this is not the case for programs using the bpf loader (see TokenProgram). Since this is a general problem, I would wait for a fix and add the security-txt for programs without `bpf-upgradeable-loader` afterwards.

[Root cause](https://github.com/solana-labs/solana/blob/7d281a8ddd7866b8d74946c47739f24f82f21e7b/explorer/src/providers/accounts/index.tsx#L157)

Reply:
Ah makes sense, yeah that can come later.